app/controllers/lettings_logs_controller.rb

```diff
@@ -51,7 +53,11 @@ def show
   def edit
     @log = current_user.lettings_logs.find_by(id: params[:id])
     if @log
-      render "logs/edit", locals: { current_user: }
+      if @log.unresolved
+        redirect_to(send("lettings_log_#{@log.form.unresolved_log_redirect_page_id}_path", @log))
```

Comment:
```suggestion
        redirect_to(send("lettings_log_#{@log.form.unresolved_log_redirect_page_id}_path", @log))
```

Can this be constructed in a helper? Or at worse `Form` like `cancel_path` is?